### DISCUSSION TITLE
twinBasic is good

### DISCUSSION THREAD

**Discussion by dmrvb** [twinbasic/twinbasic#315] [Show and tell]
TwinBASIC is now detecting several errors and bits of bad coding that VB6 has failed to detect.

An error I sometimes make is failing to assign a type to a function:

Public Function xxx(a as long, b as long)

This is bad as it goes undetected, uses a variant, and so results in slower code.

TwinBasic warns me that it is defaulting to a variant. 

Sad that VB6 never got an Option Strict.

**Comment by WaynePhillipsEA** [comment]
Please see #208 for further information.  If you have ideas for other bad practices that you'd like twinBASIC to warn about, please let me know in that issue.  Thanks!
